pkgs/development/python-modules/drf-extra-fields/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, django
+, djangorestframework
+, filetype
+}:
+
+buildPythonPackage rec {
+  pname = "drf-extra-fields";
+  version = "3.7.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-1+WLj2BDIjMyi4pkgx5Q6l0Fky08tjrej+ZsDT0hrFs=";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  build-system = [
```
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters